cmd/admin/unpause_account_test.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"context"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	blog "github.com/letsencrypt/boulder/log"
+	sapb "github.com/letsencrypt/boulder/sa/proto"
+	"github.com/letsencrypt/boulder/test"
+	"google.golang.org/grpc"
+)
+
+// mockSAPaused is a mock which records the PauseRequest it received, and
+// returns the number of identifiers as a PauseIdentifiersResponse. It does not
+// maintain state of repaused identifiers.
+type mockSAUnpause struct {
+	sapb.StorageAuthorityClient
+	regIDCounter map[int64]int64
+}
+
+func (msa *mockSAUnpause) UnpauseAccount(ctx context.Context, in *sapb.RegistrationID, _ ...grpc.CallOption) (*sapb.Count, error) {
+	if _, ok := msa.regIDCounter[in.Id]; ok {
+		msa.regIDCounter[in.Id] += 1
+	}
+
+	return &sapb.Count{Count: msa.regIDCounter[in.Id]}, nil
+}
+
+func TestUnpausingAccounts(t *testing.T) {
```

Comment:
Same suggestion here: split this into separate tests for `readUnpauseAccountFile` and `unpauseAccounts`.